jobs/service-fabrik-virtualhost-manager/templates/config/settings.yml.erb

```diff
@@ -25,6 +25,7 @@ production:
   log_level: <%= link("broker").p('log_level') %>
   enable_circuit_breaker: <%= link("broker").p('enable_circuit_breaker') %>
   http_timeout: <%= link("broker").p('http_timeout') %>
+  agent_operation_timeout: <%= link("broker").p('agent_operation_timeout') %>
```

Comment:
Not required but please check once. If it is not required then we can remove it.

Reply:
Just kept this as VhostAgent extending Agent. Though in Agent.js the config not. Used just keeping it as related to agent call.